app/Http/Controllers/UserController.php

```diff
@@ -44,9 +44,22 @@ public function displayProfile(User $userId)
 
     public function followUser(User $profileId)
     {
-        $currentUser = Auth::user();
 
+        $currentUser = Auth::user();
         $profileId->followers()->attach($currentUser->id);
+
+        $lastId= DB::table('followers')->orderBy('created_at','desc')->first()->id;
+        
+        $followedUser = $currentUser->followings->last()->name;
+
+        Activity::create([
```

Comment:
same with comments below, create `setter` functions in the model so what the controller does is just call it from the `Model Class`